app/components/forms/wizard/basic-details-step.js

```diff
@@ -139,13 +139,14 @@ export default Component.extend(FormMixin, EventWizardMixin, {
       return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
     };
     $.fn.form.settings.rules.checkMaxMinOrder = () => {
-      return $('.ui.form').form('get value', 'ticket_min_order') <= $('.ui.form').form('get value', 'ticket_max_order');
+      return parseInt($('.ui.form').form('get value', 'ticket_min_order'),10) <= parseInt($('.ui.form').form('get value', 'ticket_max_order'),10);
     };
 
     const validationRules = {
       inline : true,
       delay  : false,
       on     : 'blur',
+      on     :'change',
```

Comment:
Codacy found an issue: [Missing space before value for key 'on'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6290947)